### DISCUSSION TITLE
Save Resources

### DISCUSSION THREAD

**Discussion by MAnDumu** [UE-Explorer/UE-Explorer#23] [General]
Greetings,

(for game **Killing Floor**)
I'd like to know if it is possible to **export resources** from **.u**, such as **.ukx**, **.utx**, etc.
I'd pay for that tool :D

Regards

**Comment by EliotVU** [comment]
Hello,

UE Explorer does not currently support exporting of resources that are common for .ukx and .utx files.

Perhaps try UE Viewer? It is capable of exporting meshes and textures to their raw formats, which you can then re-import as a .utx using the Unreal Editor.
